spec/opportunity/README.md

```diff
@@ -1,3 +1,59 @@
+
+Before ordering a user would be able to explore the opportunities which are available for chosen order configuration. The opportunities are 
+represented in a FeatureCollection, with order specific attributes and values in the feature properties.
+An example json response can be found [here](../examples/tasking-api/opportunities-response.json).
+
+
+## Opportunity fields
+
+This object describes a STAC Item. The fields `id`, `type`, `bbox`, `geometry` and `properties` are
+inherited from GeoJSON.
+
+| Field Name | Type                                                                       | Description |
+| ---------- | -------------------------------------------------------------------------- | ----------- |
+| type       | string                                                                     | **REQUIRED.** Type of the GeoJSON Object. MUST be set to `Feature`. |
+| stat_version | string                                                                   | **REQUIRED.** The STAT version the Opportunity implements. |
+| stat_extensions | \[string]                                                             | A list of extensions the Opportunity implements. |
+| product_id         | string                                                             | **REQUIRED.** Product identifier. The ID should be unique and is a reference to the constraints which can be used in the constraints field. |
+| id         | string                                                                     | Provider identifier. This is not required, unless the provider tracks user requests and state for opportunities. |
+| geometry   | [GeoJSON Geometry Object](https://tools.ietf.org/html/rfc7946#section-3.1) \| [null](https://tools.ietf.org/html/rfc7946#section-3.2) | **REQUIRED.** Defines the full footprint of the asset represented by this item, formatted according to [RFC 7946, section 3.1](https://tools.ietf.org/html/rfc7946#section-3.1). The footprint should be the default GeoJSON geometry, though additional geometries can be included. Coordinates are specified in Longitude/Latitude or Longitude/Latitude/Elevation based on [WGS 84](http://www.opengis.net/def/crs/OGC/1.3/CRS84). |
+| bbox       | \[number]                                                                  | **REQUIRED if `geometry` is not `null`.** Bounding Box of the asset represented by this Item, formatted according to [RFC 7946, section 5](https://tools.ietf.org/html/rfc7946#section-5). |
+| properties | [Properties Object](#properties-object)                                    | **REQUIRED.** A dictionary of additional metadata for the Item. |
+| links      | \[[Link Object](#link-object)]                                             | List of link objects to resources and related URLs. |
+| constraints | Map<string, \[\*]\|[Range Object](#range-object)\|[JSON Schema Object](#json-schema-object)> | A map of opportunity constraints, either a set of values, a range of values or a JSON Schema. |
```

Comment:
My understanding is that this is meant to be the **input** to `/opportunities`. In that case I think it should exactly match what you have on order/README.md